kernel/src/sched.rs

```diff
@@ -411,6 +411,12 @@ impl Kernel {
                                     }
                                     process.set_syscall_return_value(res.into());
                                 }
+                                Syscall::EXIT { mode } => {
+                                    if config::CONFIG.trace_syscalls {
+                                        debug!("[{:?}] notify_panic({:?})", appid, mode);
```

Comment:
Given that we would like to use this to enable apps to signal exits in cases other than panics, I think this debug statement should change.

Reply:
Sure, I fixed that.